tests/v3_api/test_auth.py

```diff
@@ -673,6 +725,63 @@ def enable_ad_nestedgroups(username, token, expected_status=200):
     print "Enable ActiveDirectory nestedgroup request for " + username + " " + str(expected_status)
 
 
+def search_ad_users(searchkey, token, expected_status=200):
```

Comment:
@carolynhu Do not hard code expected values here and get them passed to this method as one of the parameters.
Same should be done for other search methods as well .